### PR TITLE
Components homework

### PR DESCRIPTION
**Pull request by lessarea**

### CODE REVIEW

File: src/app/courses/courses.component.html [hunk 1]
```diff
@@ -1 +1,8 @@
-<agmp-courses-list></agmp-courses-list>
+<div fxLayout="column" class="courses__container">
+  <agmp-courses-search></agmp-courses-search>
```
Comment (andrewlistopadov) [inline src/app/courses/courses.component.html:2]: I know it's one of best practises, but I would get rid of the feature component `agmp` prefix, like agmp-courses-search.
For shared components, like button or icon, it's nice to have.
But feature components hint that they are your own custom html elements.

File: src/styles.scss [hunk 2]
```diff
@@ -1 +1,34 @@
-/* You can add global styles to this file, and also import other style files */
+* {
```
Comment (andrewlistopadov) [inline src/styles.scss:1]: Please, move all the patch styles into a separate file.

File: tsconfig.json [hunk 3]
```diff
@@ -10,6 +10,8 @@
     "module": "esnext",
     "moduleResolution": "node",
     "importHelpers": true,
+    "resolveJsonModule": true,
```
Comment (andrewlistopadov) [inline tsconfig.json:13]: What do you use it for?

Comment (lessarea) [reply tsconfig.json:13]: fake data.json with courses for now, I can remove it later

Comment (andrewlistopadov) [reply tsconfig.json:13]: Okay, got you.

**Pull request merged**